server/app/configure/authentication/local.js

```diff
@@ -95,6 +96,12 @@ function mergeByProperty(arr1, arr2, prop) {
                     req.logIn(user, function (loginErr) {
                         if (loginErr) return next(loginErr);
                         // We respond with a response object that has user with _id and email.
+                        Address.create({user_id: user._id})
+                        .then(function(address){
+                            return User.findByIdAndUpdate(address.user_id, {shipping_address: address._id})
+                        });
+
+
```

Comment:
i think this needs to be chained to the Cart.create and all of the stuff below as well.